README.md

```diff
@@ -527,6 +527,21 @@ HUGGINGFACE_API_TOKEN="YOUR_HUGGINGFACE_API_TOKEN"
 sudo Xvfb :10 -ac -screen 0 1024x768x24 & DISPLAY=:10 <YOUR_CLIENT>
 ```
 
+## Event Dispatcher
+
+By default, Auto-GPT writes output to the prompt. The event dispatcher enables sending all console output to an endpoint where further actions, such as an enterprise service bus or 3rd party applications reacting to ongoing output, can take place. The event dispatcher asynchronously fires and forgets such events through the HTTP POST method.
+
+To enable the event dispatcher, set these variables in your `.env`:
+
+```bash
+EVENT_DISPATCHER_ENABLED=True
+EVENT_DISPATCHER_PROTOCOL=http
+EVENT_DISPATCHER_HOST=localhost
+EVENT_DISPATCHER_ENDPOINT=/other-microservice
+EVENT_DISPATCHER_PORT=45000
+EVENT_DISPATCHER_MUTE=True
+```
+
```

Comment:
the readme.md got refactored significantly. May want to migrate this to a more appropriate location once updated

Reply:
moved